lib/models/device.js

```diff
@@ -1867,7 +1867,10 @@ const getDeviceModel = function (deps, opts) {
 		 * // randomKey is a randomly generated key that can be used as either a uuid or an api key
 		 * console.log(randomKey);
 		 */
-		generateUniqueKey: registerDevice().generateUniqueKey,
+		generateUniqueKey: function () {
+			// Wrap with a function so that we can lazy load registerDevice
+			return registerDevice().generateUniqueKey.apply(this, arguments);
```

Comment:
I'd prefer ...arguments as it keeps the 'this' binding the same as device registry will expect

Reply:
Actually, since we previously assigned the function to the exported method, the `this` w/ this approach is exactly as before.
But since the `this` isn't relevant for the `generateUniqueKey()` changing will also communicate that to future readers of this code :)

Reply:
Tried `...arguments` but TS required an extra `Array.from` in order to allow `...`.
So I ended up doing the equivalent with `...args`.